ecf/scripts/workflow_manager/cycled/next_run.ecf

```diff
@@ -0,0 +1,59 @@
+#PBS -S /bin/bash
+#PBS -N %RUN%_next_run_%CYC%
+#PBS -j oe
+#PBS -q %QUEUE%
+#PBS -A %PROJ%-%PROJENVIR%
+#PBS -l walltime=00:30:00
+#PBS -l place=vscatter,select=1:ncpus=1:mem=1GB
+#PBS -l debug=true
+
+model=gfs
+%include <head.h>
+%include <envir-p1.h>
+
+set -x
+
+export NET=%NET:gfs%
+export RUN=%RUN%
+export CDUMP=%RUN%
+
+############################################################
+# Load modules
+############################################################
+
+module list
+
+export edate=%EDATE%
```

Comment:
Where is %EDATE% defined?  Presumably you would in the `cycled_gfs.def`, but I didn't see it in that file

Reply:
The EDATE is added at the beginning of ecflow workflow setup were user modify config.base to set EDATE. The user will enter associated commend for ecflow.

Reply:
`config.base` defines `EDATE`.  Yes this is true.
I am trying to understand where  `config.base` is being sourced before this point in this particular job to obtain `%EDATE%`.

Reply:
At this time. It is not been done atomically. It is done my enter ecflow commend at the time first setup the workflow.

Reply:
Ok.  In a follow up PR, please add a document `docs/ecflow_dev.md` to outline how a developer could setup and execute an ecflow based workflow.

Reply:
ok